subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/ExclusiveFileAccessManager.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2012 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.executer;
+
+import java.io.Closeable;
+import java.io.File;
+import java.io.RandomAccessFile;
+import java.nio.channels.FileChannel;
+import java.nio.channels.FileLock;
+import java.util.concurrent.Callable;
+
+public class ExclusiveFileAccessManager {
```

Comment:
🤔 It looks like this type could be moved into the `files` project, allowing the deletion of the duplicate `org.gradle.wrapper.ExclusiveFileAccessManager` and consolidation on a single type.

Reply:
ah, this is a good idea.